Khan/perseus
# Fix rollup build errors

## Summary:
There were a number of issues that required additional plugins and in one case I had the wrong version of a peer dep.  I also moved at least one peer dep of a wonder-blocks component into the top-level package.json.  I think we should only list peer deps of things we explicitly import as opposed to sub-deps in order to keep the package.json for each package clean.  We can audit the package.json files and the rollup output later.  NOTE: The rollup plugin for processing .less files outputs a .css file so we'll need to make sure that that's being loaded.  There's an option to have the generated bundle create the proper script tag, but I think it may be easier to get this write if we explicitly import the .css file wherever we import the perseus bundle itself.

Issue: None

## Test plan:
- yarn build
- see that all the .js files are written to the dist folders
- see that there's a .css in packages/perseus/dist
